pkg/agent/route/route_linux.go

```diff
@@ -561,6 +565,17 @@ func (c *Client) restoreIptablesData(podCIDR *net.IPNet, podIPSet, localAntreaFl
 		}...)
 	}
 	writeLine(iptablesData, iptables.MakeChainLine(antreaPostRoutingChain))
+	// The masqueraded multicast traffic will become unicast so we
+	// stop traversing this antreaPostRoutingChain for multicast traffic.
+	if c.multicastEnabled && !c.networkConfig.TrafficEncapMode.SupportsEncap() {
```

Comment:
Use `c.networkConfig.TrafficEncapMode.SupportsNoEncap()` instead of `!c.networkConfig.TrafficEncapMode.SupportsEncap()` ?

Reply:
updated